dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -226,123 +233,92 @@ public static HashMap<String, List<String>> importFile(
             int countryCodeHeaderColumn, Reader reader, String wfActionId,
             final HttpServletRequest request) throws DotRuntimeException, DotDataException {
 
-        return importFile(importId, currentSiteId, contentTypeInode, keyfields, preview,
-                isMultilingual, user, language, csvHeaders, csvreader, languageCodeHeaderColumn,
-                countryCodeHeaderColumn, wfActionId, request, null);
+        ImportResult result = importFileResult(importId, currentSiteId, contentTypeInode,
```

Comment:
again many arguments for a method

Reply:
This is related with how the `ImportUtil` already works and it is outside the scope of the ticket.